src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Filters/ValidateAntiforgeryTokenIfAuthCookieFilter.cs

```diff
@@ -73,8 +78,13 @@ protected virtual bool ShouldValidate(AuthorizationFilterContext context)
                 return false;
             }
 
+            if(_settings.DisableCsrfCheck)
```

Comment:
could we do this check before checking the context for parameters? It would save us two checks if csrf is disabled.